src/main/java/com/squareup/javapoet/JavaFile.java

```diff
@@ -131,6 +135,13 @@ private void emit(CodeWriter codeWriter) throws IOException {
       codeWriter.emit("\n");
     }
 
+    if (!staticImports.isEmpty()) {
+      for (String signature : staticImports) {
```

Comment:
The list is a left-over from the `StaticImportItem` class. Will convert it to a sorted set.